src/OVAL/oval_probe.c

```diff
@@ -329,14 +331,13 @@ int oval_probe_query_sysinfo(oval_probe_session_t *sess, struct oval_sysinfo **o
 	return(0);
 }
 
-static int oval_probe_query_criteria(oval_probe_session_t *sess, struct oval_criteria_node *cnode);
+static oval_result_t oval_probe_query_criteria(oval_probe_session_t *sess, struct oval_criteria_node *cnode, struct oval_result_criteria_node *rslt_cnode);
```

Comment:
(Personal opinion.)

In generally, I don't like "acronyms" in variable names rather than "rslt_cnode" I prefer something like "result_cnode", especially when it is not much longer. When I see "rslt" - I need to think what is it.
There are other similar variable names...